examples/guides/testing/writing-tests-using-data-providers/hello-service/tests/hello_service_test.bal

```diff
@@ -0,0 +1,61 @@
+import ballerina/test;
+import ballerina/http;
+import ballerina/io;
+
+http:Client clientEP = new("http://localhost:9090/hello");
+
+@test:Config {
+    dataProvider: "helloServiceDataProvider"
+}
+
+//[postive]This function verifies for the response of the service.
+//It asserts for the response text and the status code.
```

Comment:
Function comments should adhere to Ballerina Style Guide.

Reply:
Corrected.